tests/test_llama2.py

```diff
@@ -1,6 +1,6 @@
 import unittest
-import pychatml
 
+from pychatml.llama2_converter import Llama2
```

Comment:
Let’s call the class just Llama 2

Reply:
It clashes with the existing llama2 directory, so can't do that.